### PR TITLE
Spanish translation (finished)

### PR DESCRIPTION
**Pull request by gubatron**
As I move along through the translated items, please let me know if I should translate differently, if I have any orthographic errors, missing accents and what not.

Thanks!

### CODE REVIEW

File: features/step_definitions/select-language-steps.js [hunk 1]
```diff
@@ -42,3 +46,12 @@ Then(/^I should have Japanese language set$/, async function () {
...
 });
+
+Then(/^I should have Spanish language set$/, async function () {
+  const result = await this.client.executeAsync((done) => {
+    daedalus.stores.profile.getProfileLocaleRequest.execute()
+      .then(done)
+      .catch((error) => done(error));
+  });
+  expect(result.value).to.equal('es-ES');
+});
```
Comment (jmitchell) [inline features/step_definitions/select-language-steps.js]: Additions to this file are only needed if they are used in feature tests like https://github.com/input-output-hk/daedalus/blob/master/features/select-language.feature

The feature test I linked currently verifies it's possible to select the Japanese language and "set" it, given that you start with no language set. A possible improvement would be adding a test to demonstrate switching from having language A set to language B set via the selection UI works as expected.

Comment (gubatron) [reply features/step_definitions/select-language-steps.js]: @jmitchell If I created this new test for Spanish, would it be correct to
 1. rename `select-language.feature` into `select-japanese-language.feature`
 2. have the new test called `select-spanish-language.feature`
?

**Comment by gubatron** [comment]
It's been a while since I started this translation PR, today I finished, but what I translated was based on 186 commits ago. I'm going to write a script/tool to check missing and un-needed translation entries, the input being the base translation file (english) and the other translation file to check, the output should be a list of missing keys and a list of unused keys.

**Comment by gubatron** [comment]
[Here's the tool](https://github.com/input-output-hk/daedalus/pull/841) to see how far behind and what unused translation entries exist in a particular translation file with respect to another one.

**Comment by gubatron** [comment]
Locales for Spanish 100% complete after the branch was _rebased_. 
Commits have now been squashed to not pollute the master branch's history log.

```
/usr/local/bin/node /Users/gubatron/workspace.frostwire/daedalus/scripts/i18n_check.js ../app/i18n/locales/en-US.json ../app/i18n/locales/es-ES.json

========================================================================================

Target file is up to date, no translation entries missing.


========================================================================================

Target file has no unused left over translation entries.
```

**Comment by gubatron** [comment]
Note: This PR does not include the terms of use translation, I'll do that one after this PR is merged.

**Comment by nikolaglumac** [comment]
Thank you for your contribution! Unfortunately, at this moment in time, we can not accept this translation as it would be hard to maintain it without internal translation/audit process.

**Pull request closed without merging**